.github/workflows/main.yml

```diff
@@ -0,0 +1,104 @@
+on:
+  push:
+    branches:
+      - master
+      - stable
+  pull_request:
+    branches:
+      - master
+name: Linux CI
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Linux build & test
+    env:
+      PROCESSORS: 0
+      PREFIX: "/tmp/ADALIB_DIR"
+      GPR_PROJECT_PATH: /tmp/ADALIB_DIR/share/gpr
+    steps:
+      - name: Get langkit
+        uses: actions/checkout@v2
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Install python dependencies
+        run: |
+            pip install -r REQUIREMENTS.dev
+            pip install jsonschema
+            pip install mypy==0.780
+            python setup.py install
+            pip install flake8
```

Comment:
Shouldn’t jsonschema, mypy and flake8 move to `REQUIREMENTS.dev`? Also I think we should replace `pyhton setup.py install` with `pip install .`.

Reply:
mypy and flake8 indeed, I'm not even sure what jsonschema is doing here, I think it's necessary for LAL, not langkit